wagtail_localize/models.py

```diff
@@ -395,7 +396,7 @@ def create_or_update_translation(self, locale, user=None, publish=True, copy_par
             created = True
 
         # Copy synchronised fields
-        for field in getattr(translation, 'translatable_fields', []):
+        for field in get_translatable_fields(original.__class__):
             if field.is_synchronized(original):
                 # TODO: Use Django to set the field so the attname is correct
```

Comment:
Is this TODO still required?

Reply:
Yep, but it will go away in https://github.com/wagtail/wagtail-localize/pull/179